scylla/src/transport/session.rs

```diff
@@ -1311,13 +1349,18 @@ impl Session {
             ..Default::default()
         };
 
-        self.run_query(
+        match self.run_query(
             info,
             &config,
             |node: Arc<Node>| async move { node.random_connection().await },
             do_query,
         )
-        .await
+        .await? {
+            RunQueryResult::IgnoredWriteError => Err(QueryError::UserLogicError(
+                "User's retry policy has made the driver ignore schema's agreement query, which is a user's bug.")
```

Comment:
Let's not be overeager here. Users are technically free to do that, even if just for the purpose of testing what happens when schema agreement is ignored. No need to blame the user

Reply:
Do you suggest only changing the error message, or also the variant (to which?), or even a solution different than returning an error here?

Reply:
@psarna

Reply:
I wanted UserLogicError to be gone, and it is, and I also see that the  `, which is a user's bug` postfix is also gone. Looks good to me